DirectProgramming/DPC++FPGA/Tutorials/Features/latency_control/src/latency_control.cpp

```diff
@@ -0,0 +1,123 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <numeric>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+
+using BurstCoalescedLSU = ext::intel::experimental::lsu<
+    ext::intel::experimental::burst_coalesce<true>,
+    ext::intel::experimental::statically_coalesce<false>>;
+
+constexpr size_t kNum = 5;
```

Comment:
You can remove this constexpr variable and assign 5 directly to size at line 94

Reply:
Sure, will do.